Phases/Assessments/Phase_1.md

```diff
@@ -0,0 +1,37 @@
+# Phase 1 Assessment
+
+The phase 1 assessment will test your ability to write basic JavaScript programs for the Node.js platform as well as your fundamental SQL knowledge.
+
+You will _not_ be assessed on your HTML, CSS, or other web-specific skills.
+
+## Skills Assessed
+
+- Programming (Beginner Level)
+- JavaScript (Beginner Level)
+- Node.js (Beginner Level)
+- Testing (Beginner Level)
+- SQL (Beginner Level)
+
+Some examples of specific things you _may_ be required to do as part of the assessment (this is **not** an exhaustive list):
+
+- Manipulate Arrays, Strings, and Objects in JavaScript
+- Use conditionals and iteration to solve a problem
+- Pull values out of a JSON object
+- Write tests for your code with Mocha and Chai
+- Use synchronous functions from the Node.js standard library
+- Read and write to files from a Node.js program
+- Design a relational database schema
+- Write SQL to query a small PostgreSQL database (1-4 tables)
```

Comment:
Since there are an infinite number of things not included covered in any assessment, would prefer if we just specified exactly what _is_ in phase 1 rather than what's not. :)